README.md

```diff
@@ -45,6 +45,28 @@ export default {
 }
 ```
 
+### Runtime Config
+
+The use of [runtime config](https://nuxtjs.org/guide/runtime-config) is mandatory in case of using environment variables in production, otherwise, the values will be hard coded during build and won't change.
```

Comment:
I think wording may be little bit misleading here. It is not mandatory but required if we want to use dynamic environment variables after the build.

```suggestion
You can use [runtime config](https://nuxtjs.org/guide/runtime-config) if need to use dynamic environment variables in production. Otherwise, the options will be hardcoded during the build and won't be read from `nuxt.config` anymore.
```